packages/expo-doctor/src/checks/__tests__/InstalledDependencyVersionCheck.test.ts

```diff
@@ -0,0 +1,58 @@
+import spawnAsync from '@expo/spawn-async';
+
+import { InstalledDependencyVersionCheck } from '../InstalledDependencyVersionCheck';
+
+const asMock = <T extends (...args: any[]) => any>(fn: T): jest.MockedFunction<T> =>
+  fn as jest.MockedFunction<T>;
+
+describe(InstalledDependencyVersionCheck, () => {
+  let check;
+  beforeEach(() => {
+    check = new InstalledDependencyVersionCheck();
+  });
```

Comment:
Move into the describe to reduce the possible scope of access. Could also just inline `const check = ...` in each test to reduce any chance of the tests going awry.